sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -22,7 +22,7 @@ To use Azure services, including Azure Event Hubs, you'll need a Microsoft Azure
 
 To interact with Azure Event Hubs, you'll also need to have an Event Hub available.  If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [creating an Event Hub using the Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create).  There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create an Event Hub.
```

Comment:
I'd reorganize the prerequisites into bullet points to make them more actionable. I feel like the long blocks of text without any bold etc makes it very skippable (bold for the actual items could also be nice followed by description on how to get it).

Reply:
I played with this a bit and couldn't come up with a list form that conveyed the information that I believe is helpful;  decided to keep it as is.

Reply:
in that case how about making the actual elements needed bold (subscription, existing Event Hub)? And make sure the button and it's text is a part of the second paragraph to make it clear that's one way of doing it and not a separate item that is needed

Reply:
I included the bold suggestion (thanks!) but am limited in formatting options for the button without dropping down to HTML-in-Markdown.    I wasn't able to get things to a point where I was happy with the formatting without the button existing on its own, and bringing the button prompt into the second paragraph made things too busy and less clear in my opinion.   I think I'm going to leave as-is for now and keep a note to look with fresh eyes tomorrow and see if inspiration hits me for further revisions.   

New PR:  https://github.com/Azure/azure-sdk-for-net/pull/6700